modules/getting-started-cli-examining-pod.adoc

```diff
@@ -13,12 +13,11 @@ You can view the pods in your cluster and to determine the health of those pods
 
 .Prerequisites
 
-* You must have access to an {product-title} cluster.
-* You must have installed the OpenShift CLI (`oc`).
-* You have a deployed image.
+* You have deployed the `parksmap` front-end application.
 
 .Procedure
-. To list all pods with node names, enter the following command:
+
```

Comment:
I assume the steps here can be done in any order. Just wanted to confirm since they're in an unordered list.

Reply:
Yes that's true. It's just an assortment of what you can do to explore the pod